File: sharedModule/src/main/java/ru/nekit/android/qls/window/WindowContentViewHolder.java
package ru.nekit.android.qls.window;

import android.content.Context;
import android.support.annotation.IdRes;
import android.support.annotation.LayoutRes;
import android.support.annotation.NonNull;
import android.view.View;

import ru.nekit.android.qls.utils.ViewHolder;
import ru.nekit.android.shared.R;

public class WindowContentViewHolder extends ViewHolder {

    public WindowContentViewHolder(@NonNull Context context, @LayoutRes int layoutId) {
        super(context, layoutId);
    }

    @NonNull
    View getCloseButton() {
        return view.findViewById(getCloseButtonId());
    }

    @IdRes
    protected int getCloseButtonId() {
        return R.id.btn_close;
    }
}

File: sharedModule/src/main/java/ru/nekit/android/qls/window/WindowContentViewHolder.kt
package ru.nekit.android.qls.window

import android.content.Context
import android.support.annotation.IdRes
import android.support.annotation.LayoutRes
import android.view.View
import ru.nekit.android.shared.R
import ru.nekit.android.utils.ViewHolder

open class WindowContentViewHolder(context: Context, @LayoutRes layoutId: Int) : ViewHolder(context, layoutId) {

    internal val closeButton: View
        get() = view.findViewById(closeButtonId)

    @get:IdRes
    protected open val closeButtonId: Int
        get() = R.id.btn_close
}